query_server/spi/src/query/logical_planner.rs

```diff
@@ -812,7 +812,7 @@ impl FileFormatOptionsBuilder {
     pub fn build(self) -> FileFormatOptions {
         FileFormatOptions {
             file_type: self.file_type.unwrap_or(FileType::CSV),
-            delimiter: self.delimiter.unwrap_or(','),
+            delimiter: self.delimiter.unwrap_or_else(|| ','),
```

Comment:
The char type is too short to add a closure to get it, and it's an anonymous constant.